src/main/java/net/pms/formats/WebTT.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Universal Media Server, for streaming any medias to DLNA
+ * compatible renderers based on the http://www.ps3mediaserver.org.
+ * Copyright (C) 2012 UMS developers.
+ *
+ * This program is a free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; version 2
+ * of the License only.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package net.pms.formats;
+
+public class WebTT extends SubtitleFile {
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public Identifier getIdentifier() {
+		return Identifier.WebTT;
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public String[] getSupportedExtensions() {
+		return new String[] { "wtt" };
```

Comment:
Btw, I got "WEBTT" for ```DefaultRenderer.conf````  - so it's wrong there and in some renderer configurations as well...